easybuild/easyconfigs/p/Python/Python-3.10.8-GCCcore-12.2.0.eb

```diff
@@ -385,19 +383,19 @@ exts_list = [
         'checksums': ['9e77e941042ad3a4cc305dcdf2b2dec1aec2fbe3dd9015d2698ad02b173006d1'],
     }),
     ('pastel', '0.2.1', {
-        'source_tmpl': SOURCE_WHL,
+        'source_tmpl': '%(name)s-%(version)s-py2.py3-none-any.whl',
         'checksums': ['4349225fcdf6c2bb34d483e523475de5bb04a5c10ef711263452cb37d7dd4364'],
     }),
     ('crashtest', '0.3.1', {
-        'source_tmpl': SOURCE_PY3_WHL,
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
```

Comment:
please stick to using `SOURCE_PY3_WHL` and `SOURCE_WHL` (see above) - also a side-effect of using `--inject-checksums` which we haven't fixed yet...